lib/actions/preload.coffee

```diff
@@ -179,7 +179,7 @@ module.exports =
 			signature: 'pin-device-to-release'
 			boolean: true
 			description: 'Pin the preloaded device to the preloaded release on provision'
-			alias: 'p'
```

Comment:
Cool -- does that work?! :-) I didn't know that the alias could have more than one character.

Reply:
Affirmative, just checked to be sure :)